guides/common/modules/con_how-to-synchronize-content-using-export-and-import.adoc

```diff
@@ -13,3 +13,11 @@ For more information, see xref:Using_Upstream_Server_to_Sync_Content_View_Versio
 * You sync a single repository.
 This can be useful if you use the Content-View syncing approach, but you want to sync an additional repository without adding it to an existing Content View.
 For more information, see xref:synchronizing-a-single-repository_{context}[].
++
+[NOTE]
+====
+When you are unable to match upstream and downstream {Project} versions, you can use:
+
+. Syncable exports and imports.
```

Comment:
@parthaa What was the first release that had syncable imports? Maybe this shouldn't be cherry-picked before then.

Reply:
Actually I believe it was 6.13. This bug was initially asked for 6.12 so if we can use something different for 6.12, I can create a separate PR for the 6.12 branch.